src/containers/SendContainer/SendContainer.js

```diff
@@ -115,7 +115,7 @@ const SendContainer = (props: Props) => {
     <ContainerWithHeader
       headerProps={{ centerItems: [{ title: customScreenTitle || t('transactions.title.sendScreen') }] }}
       footer={<SendFooter {...footerProps} />}
-      minAvoidHeight={800}
+      minAvoidHeight={900}
```

Comment:
Could you explain why have you tweaked this value? Both previous value and this one seem to be quite random.

Reply:
Indeed both previous and current value is random. ContainerWithHeader checks this value internally with screen height and minAvoidHeight is used for smaller screens if the screen is small then leave footer area locked at the bottom and otherwise if screen is big enough then lock footer items above keyboard. 
Will revert the changes and will find another approach.